django_tables2/columns/filecolumn.py

```diff
@@ -32,11 +32,19 @@ class FileColumn(Column):
         verify_exists (bool): attempt to determine if the file exists
             If *verify_exists*, the HTML class ``exists`` or ``missing`` is
             added to the element to indicate the integrity of the storage.
+        text (str): If set, this value will be used to render the
+            text inside the link instead of the file's basename.
```

Comment:
The other PR has the benefit of allowing `text` to be a callable, It's better to have that consistency.